upgrade_tests/test_scenarios/test_capsule.py

```diff
@@ -29,7 +28,7 @@
 )
 
 
-class Scenario_capsule_sync(TestCase):
+class Test_Capsule_Sync():
```

Comment:
I would like to have them all in small caps. But not sure if pytest picks class as test that starts with Test/test.

If not, please update them to all caps

Reply:
If we are inherting the TestCase Class from unittest2 , Pytest will pick the class.
But if we are not Inherting the TestCase Class from unittest2, pytest needs the Class name to start with "Test"